### PR TITLE
Tableau unpack gps data into altitude, precision, latitude and longitude columns

### LINKED ISSUE #1777: Upgrade WDC connector to support Tableau v2
### Environmental Information
production, stage, local

### Expected behavior
The open-data endpoint should be able to import data to tableau for visualization.

### Additional Information
A WDC (Web Data Connector) is a HTML page with JavaScript code that connects to web data (for example, by means of a REST API), converts the data to a JSON format, and passes the data to Tableau.

The current connector was created using WDC version 1.x. This connector does not work with newer versions of tableau.

To support for newer versions of tableau we could
1. Upgrade the connector to support WDC v2.x as explained [here](http://tableau.github.io/webdataconnector/docs/wdc_upgrade)

Unfortunately, there might be a need to keep the version 1.x connectors, i.e Not modify the open-data endpoint, if we hope to support earlier versions of Tableau.

2. Alternatively, based on discussions held on this issue: onaio/onadata#1296, I believe that we could use the same connector, but define our schema not only from the XForm spec, but also from the OSM Data. However this is an untested approach at the moment, but could maybe help us maintain support for all versions of tableau?

I feel this issue could be handled adequately by implementing the 2nd option. Are there any reservations I might have missed. Hoping to get people's thoughts on this
Cc: @ukanga @pld @ivermac @lincmba @DavisRayM

### LINKED ISSUE #1877: Unpack the geofield column to the specific columns (Longitude, Latitude, Altitude, Precision) with the Tableau connector
### Environmental Information

- Operating System:
- Onadata version:
- Python version:

### Problem description
Unable to create map charts on Tableau from data pushed to Tableau with the Tableau connector.

### Expected behavior
The geofield column (a space-separated list of the location information) should be unpacked to the respective columns i.e Longitude, Latitude, Altitude, Precision before being pushed to Tableau. Currently, the tableau connector seems not to pull these values from the geofield. These columns are null hence not appearing on the map in tableau

### Steps to reproduce the behavior
Upload a form with a location question, submit a few records then add a tableau app, from your tableau, Use the WDC to get the data. 

### Additional Information
image (9)
image (8)

_Logs, [related issues](github.com/onaio/onadata/issues), weird / out of place occurrences, local settings, possible approach to solving this..._

### PR DESCRIPTION
**Pull request by WinnyTroy**
### Steps taken to verify this change does what is intended

- [x] Updated tests
- [ ] QA

### Side effects of implementing this change
None

Closes #1877 
Closes #1777

### CODE REVIEW

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- onadata/apps/api/viewsets/open_data_viewset.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6061910&bid=19854554)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- onadata/apps/api/viewsets/open_data_viewset.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6061910&bid=19854554)

File: onadata/apps/api/tests/viewsets/test_open_data_viewset.py [hunk 1]
```diff
@@ -435,10 +434,17 @@ def test_tableau_get_repeat_data(self):
...
         self.assertEqual(
             row_data[0]['children_2__childs_age'],
             5)
+        # Test that gps data is unpacked for Tableau appropriately.
+        self.assertEqual(
+            row_data[0]['_gps_latitude'],
+            '-1.2625621')
+        self.assertEqual(
+            row_data[0]['_gps_longitude'],
+            '36.7921711')
```
Comment (DavisRayM) [inline onadata/apps/api/tests/viewsets/test_open_data_viewset.py:450]: Should we also assert that the `altitude` and `precision` values are correct ?

**Review by DavisRayM — COMMENTED** [review]
LGTM

**Review by ivermac — APPROVED** [review]

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- onadata/apps/api/viewsets/open_data_viewset.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6061910&bid=19854554)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- onadata/apps/api/viewsets/open_data_viewset.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6061910&bid=19854554)

**Review by DavisRayM — APPROVED** [review]

**Pull request merged**